accounts/models.py

```diff
@@ -9,9 +10,12 @@ class Gender(models.TextChoices):
 
 
 class UserAccount(models.Model):
-    user_name = models.CharField(max_length=30)
-    password = models.CharField(max_length=30)
-    gender = models.CharField(max_length=2, choices=Gender.choices, default=Gender.UNSPECIFIED, blank=True, null=True)
+    user = models.OneToOneField(User, related_name='useraccount', on_delete=models.CASCADE, blank=True, null=True)
+    gender = models.CharField(max_length=2, choices=Gender.choices, default=Gender.UNSPECIFIED)
     contact_number = models.CharField(max_length=12, blank=True, null=True)
-    user_birth_date = models.DateField()
-    user_email = models.EmailField(max_length=254, null=True)
+    birth_date = models.DateField(null=True, blank=True)
+    # future addition: email verification
```

Comment:
remove the lines in comments OR add a TODO task

Reply:
Changed it, but I think I need to put that in a different PR so I removed the commit for now